nltk/tokenize/casual.py

```diff
@@ -262,16 +270,34 @@ def tokenize(self, text):
                               x.lower()), words))
         return words
 
+######################################################################
+# Normalization Functions
+######################################################################
+
+def reduce_lengthening(text):
+    '''
+    Replace character sequences of length 3 or greater with sequences of length 3.
```

Comment:
I couldn't understand what you were talking about until I read the code. Please say "repeated character sequences", and add a doctest example.